includes/people-functions.php

```diff
@@ -13,7 +13,7 @@
  */
 function modify_people_post_type_args( $args ) {
 	$args['rewrite'] = array(
-		'slug'       => 'experts',
+		'slug'       => 'expert',
```

Comment:
Can you please explain what was the issue that prevent us using the experts?

Reply:
The template routing within WordPress is conflicting because our page is named `experts` and the custom post type slug is name `experts`. Whenever a user browses to `/experts/page/2/` instead of matching it with the _page_ and then turning `page` and `2` into parameters, it's trying to find a people (expert) post named with the URL `page/2`, not finding it and returning a 404.